### PR TITLE
[Bug] Fix crash when creating a new feature

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
* Fixes a bug when adding a new feature
 * Menu icon tint
 * Hide missing feature from settings

@gino-m  PTAL?

### CODE REVIEW

**Comment by scolsen** [comment]
@shobhitagarwal1612 do you have steps to reproduce? I can't seem to trigger the crash.

File: gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java [hunk 1]
```diff
@@ -132,7 +132,12 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
     viewModel.getBottomSheetState().observe(this, this::onBottomSheetStateChange);
     viewModel.getOverlappingFeatures().observe(this, this::showFeatureSelector);
     viewModel.getOpenDrawerRequests().as(autoDisposable(this)).subscribe(__ -> openDrawer());
-    viewModel.getAddFeatureResults().as(autoDisposable(this)).subscribe(this::onFeatureAdded);
+    viewModel
+        .getAddFeatureResults()
+        .subscribeOn(schedulers.io())
```
Comment (gino-m) [inline gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java:137]: As a general practice, we've been adding `subscribeOn` as far upstream as possible, ie as close to the IO or other async operation as possible to make it clearer why we're subscribing on IO thread. Would you be able to move the `subscribeOn` into the stream's definition in  `HomeScreenViewModel`?

Comment (shobhitagarwal1612) [reply gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java]: Done

**Review by gino-m — APPROVED** [review]
LGTM. Thanks again for the quick fix 🩹 !

**Pull request merged**